doc/dev/GETTING_STARTED.md

```diff
@@ -0,0 +1,159 @@
+# Getting started guide
+
+This is a getting started guide, aimed to people that are not used
+to C++ programming, configuring and compiling. If you are already used
+to such processes, you may be looking for our build guide.
+
+This guide will help you to compile your own version of F3D with required dependencies.
+Optional dependencies or covered in the build guide.
+
+ - If you are using [Linux]
+ - If you are using [Windows]
+ - If you are using [macOS]
+
+## Linux
+
+Note: The following steps were tested with Ubuntu 22 and ArchLinux but
+should work for other OSes as stated, as long as listed packages are available.
+
+### Install dependencies
+
+First make sure your system is up-to-date,
+then install the required dependencies using the terminal.
+
+#### Debian/Ubuntu/Mint
+
+```
+sudo apt install build-essential git git-lfs cmake libvtk9-dev
+```
+
+#### Fedora/Centos/RedHat
+
+```
+sudo yum install make automake gcc gcc-c++ kernel-devel git git-lfs cmake vtk
+```
+
+#### Arch Linux
+
+```
+sudo pacman -S base-devel git git-lfs cmake vtk
+```
+
+### Compile F3D
+
+Open a terminal.
+
+```
+mkdir f3d
+cd f3d
+mkdir build
+git clone https://github.com/f3d-app/f3d src
+cd build
+cmake ../src
+make
```

Comment:
For beginners you could skip the whole out-of-tree propriety to simplify a bit
```suggestion
git clone https://github.com/f3d-app/f3d
cd f3d
mkdir build
cd build
cmake ..
make
```
But I get the argument of doing it right from the start too so not a big deal either way

Reply:
I cant suggest in source build dir for a software you want to modify or contribute to.